subcell_pipeline/simulation/cytosim/run_cytosim_compression_batch_simulations.py

```diff
@@ -0,0 +1,171 @@
+# %% [markdown]
+"""
+# Run Cytosim compression simulations
+
+Notebook contains steps for running Cytosim simulations in which a single actin
+fiber is compressed at different compression velocities.
+
+This notebook provides an example of running a simulation series in which there
+are multiple conditions, each of which need to be run for multiple replicates.
+For an example of running a simulation series for a single condition for
+multiple replicates, see `run_cytosim_single_fiber_batch_simulations.py`.
+
+- [Define simulation conditions](#define-simulation-conditions)
+- [Generate configs from template](#generate-configs-from-template)
+- [Define simulation settings](#define-simulation-settings)
+- [Create and run jobs](#create-and-run-jobs)
+"""  # noqa: D400, D415
+
+# %%
+import getpass
+import sys
+from datetime import datetime
+
+from subcell_pipeline.simulation.run_batch_simulations import (
+    check_and_save_job_logs,
+    generate_configs_from_template,
+    register_and_run_simulations,
+)
+
+# %% [markdown]
+"""
+Template generator uses the Preconfig class from Cytosim. Append the path to
+`preconfig.py` from a copy of the `Cytosim` repository, or download a copy into
+this location.
+"""
+
+# %%
+sys.path.append("/home/jessica/Documents/Cytosim/python/run/")
```

Comment:
Any way to avoid this line with the local path? Specifying the cytosim path in a `.env` file at the top level directory might work.

Reply:
Good call, I had my path since I used it, but I can change to a generic one